db/migrate/20200303075727_create_listing_blocked_dates.rb

```diff
@@ -0,0 +1,10 @@
+class CreateListingBlockedDates < ActiveRecord::Migration[5.2]
+  def change
+    create_table :listing_blocked_dates do |t|
+      t.references :listing
```

Comment:
History shows that it's very useful to have community_id in each table. Let's add it here too, even though it's not technically needed.